contracts/NitroAdjudicator.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+import './ForceMove.sol';
+
+contract AssetHolder {
+    // abstraction of the parts of AssetHolder that we need
+    function setOutcome(bytes32 channel, bytes32 outcomeHash) external;
+}
+
+contract NitroAdjudicator is ForceMove {
+    function pushOutcome(
+        bytes32 channelId,
+        uint256 turnNumRecord,
+        uint256 finalizesAt,
+        bytes32 stateHash,
+        address challengerAddress,
+        bytes memory outcome,
```

Comment:
I wonder if there is a good way to define the `outcome` parameter as a struct instead of generic bytes.